.github/ISSUE_TEMPLATE/feature-request.yml

```diff
@@ -1,5 +1,5 @@
 name: Feature Request
-description: Please add feature suggestions related to TEMPLATE_ADDON here.
+description: Please add feature suggestions related to DateTimeRangePicker here.
```

Comment:
FlowingCode/TechnicalDocs#91
https://github.com/FlowingCode/TechnicalDocs/tree/master/addons#naming-convention

Already mentioned in https://github.com/FlowingCode/DateTimeRangePicker/pull/1#discussion_r2091703389